public/stylesheets/main.css

```diff
@@ -181,6 +181,37 @@ input[type="range"]::-webkit-slider-thumb {
     text-align: left;
 }
 
```

Comment:
The below changes are almost identical to each other and to the .hour-tick class. Consider merging all of the commonalities into a .tick class and applying that class to all of the relevant elements.